homeassistant/components/dnsip/config_flow.py

```diff
@@ -0,0 +1,169 @@
+"""Adds config flow for dnsip integration."""
+from __future__ import annotations
+
+import asyncio
+import contextlib
+from typing import Any
+
+import aiodns
+from aiodns.error import DNSError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_HOSTNAME,
+    CONF_IPV4,
+    CONF_IPV6,
+    CONF_RESOLVER,
+    CONF_RESOLVER_IPV6,
+    DEFAULT_HOSTNAME,
+    DEFAULT_NAME,
+    DEFAULT_RESOLVER,
+    DEFAULT_RESOLVER_IPV6,
+    DOMAIN,
+)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOSTNAME, default=DEFAULT_HOSTNAME): cv.string,
+    }
+)
+
+
+async def async_validate_url(
+    url: str, resolver_ipv4: str, resolver_ipv6: str
```

Comment:
Is this a url or a host name?

Reply:
It's a hostname. Cannot understand why I wrote url to begin with but now it's changed in all places to hostname.